config.yaml

```diff
@@ -1,6 +1,9 @@
 useFixtures: false
-
 elasticsearch:
-  host: 'https://search-talktofrank-elastic-alpha-rqcui3cnc7qs3i2ns5glqw4m7i.eu-west-2.es.amazonaws.com/'
+ host: 'https://search-talktofrank-elastic-alpha-rqcui3cnc7qs3i2ns5glqw4m7i.eu-west-2.es.amazonaws.com/'
+ drugResultCount: 3
+ suggestResultCount: 5
+ phraseResultCount: 3
+ showESResult: false
   amazonES:
-    region: 'eu-west-2'
\ No newline at end of file
+    region: 'eu-west-2'
```

Comment:
Sorry, should have said, the above can be replaced with

`
elasticsearch:
  host: 'https://search-talktofrank-elastic-alpha-rqcui3cnc7qs3i2ns5glqw4m7i.eu-west-2.es.amazonaws.com/'
  amazonES:
    region: 'eu-west-2'
`

the other values are no longer required (they were removed in the es-aws-credentials branch) & the formatting is incorrect, amazoneES should sit under elasticsearch, not showESResult as it is above.

Reply:
I LOVE IT! Doing it now